run.sh

```diff
@@ -14,7 +13,7 @@ prepare () {
     echo
     echo "Example: $0 ./api-results/"
     echo "Example: $0 ./api-results/ execute_model"
-    echo "Example: $0 ./api-results/ execute_api"
```

Comment:
Removing execute_api (the predecessor to execute_api_v2) which isn't used and no longer works (the `run.py api generate-api` command it calls doesn't exist).